test/controllers/api/v2/systems_controller_test.rb

```diff
@@ -244,5 +226,20 @@ def test_product_content
       assert_response :success
       assert_template 'api/v2/systems/product_content'
     end
+
```

Comment:
Will need to remove loading of the fake search service in this test class as well.